src/Raven.Studio/typescript/viewmodels/database/tasks/migrateRavenDbDatabase.ts

```diff
@@ -132,6 +137,47 @@ class migrateRavenDbDatabase extends viewModelBase {
             })
             .always(() => this.spinners.migration(false));
     }
+    compositionComplete() {
+        super.compositionComplete();
+
+        popoverUtils.longWithHover($("#scriptPopover"),
+            {
+                content:
+                    "<div class=\"text-center\">Transform scripts are written in JavaScript </div>" +
+                        "<pre><span class=\"token keyword\">var</span> id = doc[<span class=\"token string\">'@metadata'</span>][<span class=\"token string\">'@id'</span>];<br />" +
+                        "<span class=\"token keyword\">if</span> (id === <span class=\"token string\">'orders/999'</span>)<br />&nbsp;&nbsp;&nbsp;&nbsp;" +
+                        "<span class=\"token keyword\">throw </span><span class=\"token string\">'skip'</span>; <span class=\"token comment\">// filter-out</span><br /><br />" +
+                        "<span class=\"token keyword\">this</span>.Freight = <span class=\"token number\">15.3</span>;<br />" +
+                        "</pre>"
+            });
+
+        this.initInlineEdit();
```

Comment:
Why do you need inline edit here? What is edited inline on this view?

Reply:
done